_publications/paper-08.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "International Analysis of Electronic Health Records of Children and Youth Hospitalized With COVID-19 Infection in 6 Countries"
```

Comment:
This looks like a duplicate of `paper-04`

Reply:
I agree! But paper-04 was not listed in the main index file. I am removing paper-08 and updating the numbers in the main file.